packages/polyfill/source/Document.ts

```diff
@@ -82,7 +91,7 @@ export function createElement<T extends Element>(
   } else if (lowerName === 'template') {
     element = new HTMLTemplateElement() as any;
   } else {
-    const CustomElement = document.defaultView!.customElements.get(name);
+    const CustomElement = ownerDocument.defaultView.customElements.get(name);
```

Comment:
Why don't we pass `name` to the constructor?

Reply:
I'm not sure. It was the behaviour that is implemented.